build.cmd

```diff
@@ -1,7 +1,7 @@
 @echo off
 cls
 
-.paket\paket.bootstrapper.exe
+.paket\paket.bootstrapper.exe 3.0.0-alpha040
```

Comment:
can't at the moment https://github.com/fsprojects/Paket/issues/1464 :(
easy fix later